### ISSUE TITLE
"Can't open this page" error on all tabs on Pixel 7 Pro March 2023 Update

### ISSUE THREAD

**Issue by fireph** [bromite/bromite#2600]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser (not about the website, building Bromite, F-Droid or anything else).

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

108.0.5359.156

### Device architecture

arm64-v8a

### Android version

13

### Device model

Pixel 7 Pro

### Changed flags

no flags changed

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

Yes, I can't even open chrome://crashes (I get the same "Can't open this page" error).

### Describe the bug

On a Pixel 7 Pro on the March 2023 update, opening any tab will result in the tab crashing with error message "Can't open this page" (it does not crash the entire browser).

### Steps to reproduce the bug

Try to load any website in a tab on bromite on Pixel 7 Pro on March 2023 update (I can only assume that other Pixel devices have the same issue, but don't have any to test with).

### Expected behavior

The tab should load the content rather than crashing

### Screenshots

![Screenshot_20230313-213546](https://user-images.githubusercontent.com/443370/224905304-167e3aca-e533-4740-8cbe-4f96e8335084.png)

**Comment by cvnxbver** [comment]
I am having the same problem. Any updates?

**Comment by Nogooduser** [comment]
Duplicate of issue #2504. This has been a bug since the QPR2 beta and hasn't been fixed yet. 

QPR3 beta also has the same issue btw.

**Comment by PushaG** [comment]
Same problem here on Pixel 7, also there have been no updates for almost 4 months now, is project abandoned?

**Comment by Sanjay451** [comment]
Facing same issue on Pixel6a after the March update

**Comment by codingCoffee** [comment]
Facing the same issue here, Pixel 7 Pro

**Comment by vledder** [comment]
Same here on Pixel 6 after March update

**Comment by ragebflame** [comment]
> Same here on Pixel 6 after March update

Same for me since the March update (Pixel 7)

**Comment by rogthefrog** [comment]
Just updated a Pixel 6a to the March update and I'm experiencing the same.

**Comment by c4t-m4n** [comment]
Same here, Pixel 7 after March update. Although extremely happy with Bromite currently looking for a non-bloated, adblocked, privacy-sensitive alternative...any recommendations highly welcome.

**Comment by johnfloe** [comment]
Same here on Pixel 7 Pro. I had just started using Bromite and was very happy with it. Since it doesn't look like this is going to be solved in the near future I'm also looking for an alternative.

**Comment by codingCoffee** [comment]
Same here looking for an alternative. I'm using Chromium, from the same repo along with Adaway for adblocking for now. If anyone finds a better Bromite alternative please suggest on the thread.

**Comment by bharatmody** [comment]
I am using Vivaldi for now, not as good a Bromite but it has ad blocking built-in

**Comment by Sanjay451** [comment]
Using Brave browser now, but ads blocking is not as good as bromite. But I guess we don't have an option apart from that. 

Has anyone tried Mull browser? There's been a suggestion in this #2603 discussion

**Comment by DKB-91** [comment]
> Using Brave browser now, but ads blocking is not as good as bromite. But I guess we don't have an option apart from that. 
> 
> Has anyone tried Mull browser? There's been a suggestion in this #2603 discussion

For adblocking Brave can work better if you set it right, so does Mull.
You can change Brave's adblock settings by typing brave://adblock in the url bar and turning on/adding filters list. (Also add youf own custom rules)
There's also the option to turn Shields to "Block trackers and ads (aggresive)" if it isn't enough.

Mull is Firefox based so it comes with a lack of site process isolation.
Also it is hardened with settings based on Arkenfox, so say goodbye to dark mode for sites as that is fingerprintable.

It is however for Privacy focus on the top three browsers alongside Brave and Bromite.
It has no baked in adblocker of it's own, however it has access to the best one through Firefox's extensions, uBlock Origin.
Depending if you know how to use it in advanced mode it's easily better than Brave and Bromite for both privacy and ads, however you can also use it as just an adblocker in easy mode.

Just like Brave you might need to turn on some filters (few as possible is better) if you don't live in America or visit foreign websites.

As an Italian I live by Easylist Italy for example, for me standard Bromite blocks very little ads compared to a properly set Mull and Brave. (And Uazo's beta, but that is because he has an experimental adblocker on top of that that lets me turn on Easylist Italy.)

If you don't want to use youtube's app or install Newpipe, Mull w/uBlock Origin and Brave do block youtube's ads compared to bromite in general. (Even uazo's beta doesn't, but I just use Newpipe lmao.)

**Comment by Un1q32** [comment]
same issue on pixel 7, using firefox for android now

**Comment by Wizard-of-loneliness** [comment]
Same issue on pixel 6a, Pretty sure "March 2023" update caused this.

**Comment by SimplyAnotherAnon** [comment]
Same issue on Pixel 6 Pro as soon as I updated to The March update, using Mull until Bromite is fixed

**Comment by iMAndDev** [comment]
Got the same issue on Pixel 6a after March 2023 update

**Comment by n0r4r0n** [comment]
I need a decent alternative that is Chromium based, because I dislike Firefox's mobile UI choices so much. Does anyone know of any (besides Chromium itself)?

**Comment by Spark4000** [comment]
Mulch https://gitlab.com/divested-mobile/mulch It's probably the closest thing to Bromite. It lacks some settings though, such as image disable, timezone override, user agent, user scripts, and adblock

**Comment by ewoks** [comment]
Duplicate of #2504 and as that it should be closed. 
Please post there.

**Comment by runninghamster0** [comment]
Hi, I'm just curious because I've been using Bromite daily on my Pixel 7 with no issues. Are there any additional steps to cause this issue?
![Screenshot_20230411-125840](https://user-images.githubusercontent.com/52240313/231044877-2fcd7435-56f6-40bc-b873-18447468ed32.png)

**Comment by vladigr1** [comment]
rolled back to 94.0.4696.109 solved the issue for me

**Comment by Nogooduser** [comment]
Android 14 beta 1 seems to have fixed this issue. Not getting the error on any pages.

**Comment by quyleanh** [comment]
@welkinpc-91 Thank you for your `brave://adblock` trick. I didn't know that.
Btw, do you know any of brave folk which removes rewards, wallet. and other telemetry  functions? I just want a simple as much as possible.

**Comment by DKB-91** [comment]
> @welkinpc-91 Thank you for your `brave://adblock` trick. I didn't know that.
> Btw, do you know any of brave folk which removes rewards, wallet. and other telemetry  functions? I just want a simple as much as possible.

Not on android I'm afraid.
You can debloat it on Desktop, but not on smartphone. (You can still disable access to some Web3 content, but that's it)

Disabling Brave News and Sponsored Backgrounds is as far as you can go UI wise on phone.

**Comment by DKB-91** [comment]
@quyleanh 

Scratch that, I've been able to remove some things from the UI

![Screenshot_20230416-185455_Brave.jpg](https://user-images.githubusercontent.com/124310619/232328138-6b4e8514-88a5-4fd2-b11f-d589c12deab0.jpg)

![Screenshot_20230416-185511_Brave.jpg](https://user-images.githubusercontent.com/124310619/232328140-c5bcb202-c819-4bb2-b6c9-9875137f55f7.jpg)

Type brave://flags, set the value to disabled

![Screenshot_20230416-185729_Brave.jpg](https://user-images.githubusercontent.com/124310619/232328219-b59f4dd5-067c-4617-80cb-aef29390efc4.jpg)

![Screenshot_20230416-185738_Brave.jpg](https://user-images.githubusercontent.com/124310619/232328221-e19c3dde-385b-4bb8-9b79-85ad878d08ff.jpg)

![Screenshot_20230416-185749_Brave.jpg](https://user-images.githubusercontent.com/124310619/232328222-20ba02db-2989-49d9-b9c5-034c349f72f0.jpg)

Can't find a way to disable the Rewards and VPN button (only on desktop), also this workaround may be temporary (patched out eventually)

Would not disable any other brave unique feature as that would harm security, go just for wallet and news settings.
(Also put that google payments sandbox to default, unsure disabling it is a good idea, it's not connected to rewards I think)

Fun fact for people with eyes problems/dark mode lovers: this method can put a dark mode on your choosing on every website, and with fingerprinting set to strict, enable dark mode with returning light mode to most fingerprinting website I test to.

Type auto dark mode and find the settings with that name and choose your preferred dark mode.

Basically we can use dark mode without being fingerprinted by (some) scripts (as long as in shields the settings are set to strict), unlike Dark Reader on Firefox, pretty nice.

You can do this on any other chromium browser using chrome://flags (Bromite and uazo's Beta), but you will gain a fingerprint there, all website and repositories I've tried with uazo's beta do detect it unlike with Brave on shield strict.

**Comment by quyleanh** [comment]
@welkinpc-91 Thank you very much for your info.
After all I found out that Vivaldi is suitable for me, even when it has limit of lack of userscript, but it's fine.

**Comment by yearski** [comment]
> Hi, I'm just curious because I've been using Bromite daily on my Pixel 7 with no issues. Are there any additional steps to cause this issue?
Mine was working fine until I applied some updates yesterday, now I get "aw snap" or "can't open this page" for most, but not all pages.
Android 13; Pixel 7 Build/TQ2A.230405.003.E1

**Comment by Wizard-of-loneliness** [comment]
I just installed April 5th Android security update on my Pixel 6A, Bromite works fine now.

**Comment by Soundz1337** [comment]
> I just installed April 5th Android security update on my Pixel 6A, Bromite works fine now.

I have the same update on the same phone and its not working. Did you do something special?
